arcade/perf_info.py

```diff
@@ -179,8 +179,7 @@ def get_fps(frame_count: int = 60) -> float:
 
     start_time = _frame_times[-frame_count]
     total_time = cur_time - start_time
-    fps = frame_count / total_time
-    return fps
+    return frame_count / total_time
```

Comment:
I like the original better, as it is clear what it is doing.

Reply:
Maybe just add a comment? like `# returning fps`

Reply:
I'll revert this back to the original to be consistent with established preferences 😉